core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1506,7 +1506,7 @@ public static function addImageToTemplate($template, array $rowData, $maxWidth =
 				Metadata::VALUE_ALT => $rowData['alt'] ?? '',
 				Metadata::VALUE_TITLE => $rowData['imageTitle'] ?? '',
 				Metadata::VALUE_URL => self::replaceInsertTags($rowData['imageUrl'] ?? ''),
-				'linkTitle' => $rowData['linkTitle'] ?: '',
+				'linkTitle' => $rowData['linkTitle'] ?? '',
```

Comment:
@m-vo this correct?

Reply:
Probably. In theory someone could have set `linkTitle` to `false` or `0` previously which worked in the old code and would now result in an error. If wanted to be extra safe, we could for instance do one of these:

```suggestion
				'linkTitle' => ($rowData['linkTitle'] ?? null) ?: '',
```

```suggestion
				'linkTitle' => (string)($rowData['linkTitle'] ?? ''),
```